tests/src/core/particle/ParticleCurve.test.ts

```
@@ -0,0 +1,74 @@
+ 
+    curve.addKey(new CurveKey(0, 0.4));
+ +  expect(curve.keys.length).to.equal(3);
+ +  expect(curve.keys[0].value).to.equal(0.3);
+ + 
+    curve.removeKey(2);
+ +  expect(curve.keys.length).to.equal(2);
+ +  
+    curve.removeKey(0);
+    
+ +  expect(curve.keys.length).to.equal(1);
+ +  expect(curve.keys[0].time).to.equal(0.0);
```

ESLint

[error] Replace `+··` with `···+`

(prettier/prettier)